slack/web/client.py

```diff
@@ -573,8 +573,12 @@ def dnd_setSnooze(self, *, num_minutes: int, **kwargs) -> SlackResponse:
         kwargs.update({"num_minutes": num_minutes})
         return self.api_call("dnd.setSnooze", http_verb="GET", params=kwargs)
 
-    def dnd_teamInfo(self, **kwargs) -> SlackResponse:
-        """Retrieves the Do Not Disturb status for users on a team."""
+    def dnd_teamInfo(self, *, users: str,**kwargs) -> SlackResponse:
+        """Retrieves the Do Not Disturb status for users on a team.
+        Args:
+            users (str): Comma-seperated list of users to fetch Status for. e.g. 'U1234,W4567'
```

Comment:
If `users` can be a string or a comma-seperated list of strings the type would be something like this right?
```Python
from typing import List

def dnd_teamInfo(self, *, users: Union[str, List[str]], **kwargs) -> SlackResponse:
```

We should test this pretty heavily to ensure it works as expected.

Reply:
Ah, fair! I was thinking of it being passed as one single string that's broken up by commas rather than a list of strings. I'll need to test that. 

This may raise the question though if our situations that allow this need the same as we don't have `Union` in this:

```python
    def conversations_invite(self, *, channel: str, users: List[str], **kwargs) -> SlackResponse:
```

Reply:
@splinterific Hello, are you still interested in working on this? If so, I'm happy to take over the review here.

Reply:
If you can leave it with me until early next week @seratch , I'll let you know :)

Reply:
yeah its over to you @seratch . I won't get to it